### PR TITLE
feat: rating component and updated links

### PR DESCRIPTION
**Pull request by Muzammil-khan710**

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
Good name of class . code is clean . Everything looks good to me. You can merge it.

File: docs/components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <h2 class="t-cntr">Rating</h2>
+            <p>A rating is an evaluation or assessment of something, in terms of quality, quantity, or some combination of both .</p>
+
+            <div class="star-container">
+                <ul class="ul-ls">
+                    <li class="star orange">&#x2605;</i></li>
+                    <li class="star orange">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
```
Comment (abhishektanwar) [inline docs/components/Rating/rating.html]: Question : 
1. Why are we giving color to star like this here? nth child could be used here in combination with star-container class.
2. Why is there a closing tag for i tag?

File: docs/components/css/colors.css [hunk 2]
```diff
@@ -9,6 +9,7 @@
     --color-dark-blue: #032f3b;
     --color-dark-grey: darkgrey;
     --color-white: white;
+    --color-orange:orange;
```
Comment (abhishektanwar) [inline docs/components/css/colors.css]: Maybe using a toned down version or orange would be better here .

**Review by abhishektanwar — APPROVED** [review]
Code looks good and clean . Good to merge

File: docs/components/Rating/rating.html (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/Rating/rating.html]: 1. As i am not using JS so i have give color to the classes.. I will update when i will be adding JS to my Component Library.
2. searched online for star icon and found it . Didn't saw that i tag. Thanks for highlighting

File: docs/components/css/colors.css (hunk 2 above)
Comment (Muzammil-khan710) [reply docs/components/css/colors.css]: sure

**Pull request merged**